### DISCUSSION TITLE
rinkeby not working.

### DISCUSSION THREAD

**Discussion by thakuradi** [smartcontractkit/full-blockchain-solidity-course-py#1762] [Q&A] [answered]
there is no rinkeby token in link token .please help

**Comment by BoseSj** [comment]
Kovan, Ropsten, and Rinkeby have been deprecated. Refer to [Goerli Faucet](https://faucets.chain.link/goerli)

**Comment by thakuradi** [comment]
Thanks  Sent from Mail for Windows From: SJ BasakSent: 29 October 2022 00:23To: smartcontractkit/full-blockchain-solidity-course-pyCc: thakuradi; AuthorSubject: Re: [smartcontractkit/full-blockchain-solidity-course-py] rinkeby (Discussion #1762) Kovan, Ropsten, and Rinkeby have been deprecated. Refer to Goerli Faucet—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>

**Comment by cromewar** [comment, marked as answer]
@BoseSj is correct, those networks are deprecated and obsolete, please use Goerli instead, all the course exercises were updated to work with Goerli so just give them a look.
